### DISCUSSION TITLE
Problem installing Homebrew

### DISCUSSION THREAD

**Discussion by lauderui** [l3atbc-datadog/pushkin#84] [General] [answered]
I'm trying to install brew (following the instructions on [https://brew.sh/](https://brew.sh/)), but when I paste the command in this command

"$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

in the command line, I get the error message "illegal variable name". Does anyone know what I'm doing wrong?

**Comment by conbainbridge** [comment, marked as answer]
It looks like you may have left off the beginning of the command (/bin/bash -c):
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"`

Let us know if it still doesn't work!

**Reply by lauderui** [reply]
My bad, I cut it off when pasting into the window here (sorry!), but I pasted it correctly into the terminal. Alas, it's still not working.
![image](https://user-images.githubusercontent.com/27200899/89329722-30f6ba80-d65d-11ea-8b0c-f6e1cbe58b78.png)

**Reply by conbainbridge** [reply]
If you type into the console `which curl` what does it output?

**Reply by lauderui** [reply]
`/usr/bin/curl`

**Reply by jkhartshorne** [reply]
Can you try `$ (which curl) --version`?

**Reply by jkhartshorne** [reply]
Also `$ echo $SHELL`?

**Reply by lauderui** [reply]
`(which curl) --version` gives the message `Badly placed ()'s.`

`echo $SHELL` gives `/bin/tcsh`

**Reply by parkerrobbins** [reply]
@lauderui Try `$(which curl) --version` (with the `$`).

**Reply by jkhartshorne** [reply]
@lauderui apparently [tcsh isn't allowed](https://docs.brew.sh/Installation). Can you switch to bash or zsh? 

(I'm assuming you switched to tcsh on purpose and now how to switch to bash or zsh. If not, I can look it up for you.)

**Reply by lauderui** [reply]
OK, I'll try bash! I switched to tcsh last year because I was doing C++ stuff, but I'm not married to it :-)

Edit: Changing to bash did the trick! Thank you for your help!

**Comment by jkhartshorne** [comment]
@lauderui if this works for you, please mark @conbainbridge's response as the answer. This helps us clear the inbox.

**Reply by lauderui** [reply]
Not there yet, but I'll make sure I'll mark the response as the answer as soon as it's working!

**Comment by restmegan** [comment]
Hi @lauderui ! I just wanted to check in to see how Pushkin has been going for you. The Pushkin team is online today answering questions so if there's anything you need help with please let us know!
